src/backend/api/Fusion.Resources.Domain/Commands/Requests/InternalRequests/CreateInternalRequest.cs

```diff
@@ -132,6 +137,9 @@ private async Task<DbResourceAllocationRequest> CreateDbRequestAsync(CreateInter
                 };
 
                 dbContext.ResourceAllocationRequests.Add(item);
+                
+                await SendNotificationsAsync(new QueryResourceAllocationRequest(item));
```

Comment:
Should try to align this with notifications. So the domain layer do not need to worry about the event integration etc.. Just publish an event that you handle in the api project, or even the application project i guess..

Reply:
dispatching from api project/controller